velox/expression/tests/CastExprTest.cpp

```diff
@@ -224,6 +224,29 @@ TEST_F(CastExprTest, timestamp) {
       });
 }
 
+TEST_F(CastExprTest, timestampToString) {
+  testCast<Timestamp, std::string>(
```

Comment:
could we add a few more extreme cases here? It would be nice to test an exceedingly large (and low/negative) number to check if this correctly throws when it should, and that the behavior is consistent with what Presto Java does.

Reply:
Add test covering year 4969

For negative values - Should it be validated in constructor itself? right now Timestamp takes **int64_t**